ecs-cli/modules/commands/configure/configure_command.go

```diff
@@ -48,23 +142,30 @@ func configureFlags() []cli.Flag {
 			EnvVar: flags.AwsRegionEnvVar,
 		},
 		cli.StringFlag{
+			Name:  flags.ConfigNameFlag,
+			Value: "default",
+			Usage: fmt.Sprintf(
+				"Specifies the name of the Profile that will be stored in the configs.",
+			),
+		},
+		cli.StringFlag{
 			Name: flags.AccessKeyFlag,
 			Usage: fmt.Sprintf(
-				"Specifies the AWS access key to use. If the AWS_ACCESS_KEY_ID environment variable is set when ecs-cli configure is run, then the AWS access key ID is set to the value of that environment variable.",
+				"Specifies the AWS access key to use. Will use value of your $AWS_ACCESS_KEY_ID environment variable if it is set.",
 			),
 			EnvVar: "AWS_ACCESS_KEY_ID",
 		},
 		cli.StringFlag{
 			Name: flags.SecretKeyFlag,
 			Usage: fmt.Sprintf(
-				"Specifies the AWS secret key to use. If the AWS_SECRET_ACCESS_KEY environment variable is set when ecs-cli configure is run, then the AWS secret access key is set to the value of that environment variable.",
+				"Specifies the AWS secret key to use. Will use value of your $AWS_SECRET_ACCESS_KEY environment variable if it is set.",
```

Comment:
Specifies the AWS secret key to use. The ECS CLI uses the value of your $AWS_SECRET_ACCESS_KEY environment variable if it is set.